.github/workflows/main.yml

```diff
@@ -12,6 +12,10 @@ on:
         description: "Limit the benchmark runs to bench suites matching the comma separated names -- leave blank to run all defined experiments"
         required: false
         default: ""
+      array_max_length:
+        description: "Set length of array encoding runs (defaults to 14)"
+        required: false
+        default: ""
```

Comment:
Isn't the length independent of the selected encoding? If we want to compare two encodings we would need to run both encodings with the same length.

Reply:
I’m not sure I understand you’re concern: All encoding will run without ever length is configured here.

Reply:
That is right. My point is just that this can be used to set the length of runs for encodings other than arrays. I'm just wondering about the name, should have made that clear to begin with.

Reply:
Oh yes, I see. So by "array encoding runs" I mean "runs that benchmark the array encoding benchmarks" which of course also includes running the oopsla encoding, against which it is being benchmarked.

Any thoughts on how we should clarify the terminology here?

Reply:
`comparison_max_length` instead of `array_max_length`, or something on those lines? The source of confusion for me was actually the description, because I interpreted it as changing the length for one encoding but not the other.

Reply:
How this? db44e32?

Reply:
`ARRAY_MAX_LENGTH` on `mail.yml`?

Reply:
Oops, forgot to save some files after find/replace. Thanks for the catch.